minimum-constraints-develop.txt

```diff
@@ -33,6 +33,9 @@ coveralls==3.3.0
 # Safety CI by pyup.io
 safety==3.1.0
 safety-schemas==0.0.2
+# The following used by safety
+python-Levenshtein==0.25.1
```

Comment:
I suggest to not add the "python-Levenshtein" dependency. For the reasons, see comment above.

Reply:
Done